### PR TITLE
port Installing environment modules without root permissions page to markdown

### PR DESCRIPTION
**Pull request by orbsmiv**

### CODE REVIEW

File: docs/installing-environment-modules-without-root-permissions.md [hunk 1]
```diff
@@ -1,139 +1,116 @@
...
-    **
-    It still needs to be ported from *reStructuredText* (.rst) to *MarkDown* (.md),  
-    and you can help with that!
-    **
+## Tcl
 
-    - source: [`docs/Installing-environment-modules-without-root-permissions.rst` in `easybuilders/easybuild` repo](https://raw.githubusercontent.com/easybuilders/easybuild/develop/docs/Installing-environment-modules-without-root-permissions.rst)
-    - target: [`docs/installing-environment-modules-without-root-permissions.md` in `easybuilders/easybuild-docs` repo](https://github.com/easybuilders/easybuild-docs/tree/main/docs/installing-environment-modules-without-root-permissions.md)
+1. Go to <https://www.tcl.tk> and download the latest Tcl sources. At
+    the time of writing, the latest available Tcl version was 8.5.15,
```
Comment (orbsmiv) [inline docs/installing-environment-modules-without-root-permissions.md:10]: should this be updated to reflect the current "as of writing" version?

File: docs/installing-environment-modules-without-root-permissions.md [hunk 2]
```diff
@@ -1,139 +1,116 @@
...
+package, which requires Tcl.
 
-#. Go to the ``unix`` subdirectory of the unpacked Tcl directory, and
-   run the ``configure`` script using the ``--prefix`` option:
+## Environment Modules
 
-   .. code:: bash
+1. Download the latest source tarball for the environment modules tools
+    from <https://modules.sourceforge.net>. At the time of writing, the
+    latest available version is 3.2.10 which can be downloaded [from
```
Comment (orbsmiv) [inline docs/installing-environment-modules-without-root-permissions.md:64]: should this be updated to reflect the current "as of writing" version?

File: docs/installing-environment-modules-without-root-permissions.md (hunk 1 above)
Comment (branfosj) [reply docs/installing-environment-modules-without-root-permissions.md:10]: We'll handle any such updates separately.

File: docs/installing-environment-modules-without-root-permissions.md (hunk 2 above)
Comment (branfosj) [reply docs/installing-environment-modules-without-root-permissions.md:64]: We'll handle any such updates separately.

**Review by branfosj — APPROVED** [review]

**Pull request merged**